front/app/containers/Admin/flexible-pages/VisibleNavbarItemList/index.tsx

```diff
@@ -29,13 +29,17 @@ import messages from './messages';
 // utils
 import { isNilOrError } from 'utils/helperUtils';
 import clHistory from 'utils/cl-router/history';
-import { NAVIGATION_PATH } from '../..';
+import { NAVIGATION_PATH } from 'containers/Admin/flexible-pages';
+import useFeatureFlag from 'hooks/useFeatureFlag';
 
 const VisibleNavbarItemList = ({
   intl: { formatMessage },
 }: InjectedIntlProps) => {
-  const navbarItems = useNavbarItems();
+  const navbarItems = useNavbarItems({ standard: true });
   const pageSlugById = usePageSlugById();
+  const customizableNavbarEnabled = useFeatureFlag({
```

Comment:
Saw that useFeatureFlag now has a return type of any since the last changes, should be boolean but out of scope. Will take it to the FE CDM.